File: c/Raindrops.c
#include "raindrops.h"
#include <stdio.h>
#include <string.h>
char *convert(char result[], int nr) {
    strcpy(result, "");
    if (nr % 3 == 0)
        strcat(result, "Pling");
    if (nr % 5 == 0)
        strcat(result, "Plang");
    if (nr % 7 == 0)
        strcat(result, "Plong");
    if (!strcmp(result, ""))
        sprintf(result, "%d", nr);
    return result;
}
/*
raindrops.h
#ifndef RAINDROPS_H
#define RAINDROPS_H
char *convert(char result[], int drops);
#endif

*/

File: javascript/Raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (x) => {
  let str='';

  if (x % 3 === 0) str += 'Pling';
  if (x % 5 === 0) str += 'Plang';
  if (x % 7 === 0) str += 'Plong';

  return str || x.toString();
};
